project/Dependencies.scala

```diff
@@ -136,18 +137,17 @@ object Dependencies {
       // reactive streams tck
       val reactiveStreamsTck = "org.reactivestreams" % "reactive-streams-tck" % reactiveStreamsVersion % "test" // CC0
 
-      val protobufRuntime = "com.google.protobuf" % "protobuf-java" % "3.9.0" % "test"
+      val protobufRuntime = "com.google.protobuf" % "protobuf-java" % protobufJavaVersion % "test"
     }
 
     object Provided {
       // TODO remove from "test" config
-      // If changed, update akka-docs/build.sbt as well
-      val sigarLoader = "io.kamon" % "sigar-loader" % "1.6.6-rev002" % "optional;provided;test" // ApacheV2
+      val sigarLoader = "io.kamon" % "sigar-loader" % "1.6.6" % "optional;provided;test" // ApacheV2
```

Comment:
Isn't 1.6.6-rev002 newer than 1.6.6? Let's add a comment about that though.

Reply:
`sbt-updates` decided 1.6.6 was the latest version, but it seems 1.6.6-rev002 is latest version.
I'll revert it.